### PR TITLE
fix: Skip prune shrinkwrap if no "npm-shrinkwrap.json"

### PR DESCRIPTION
**Pull request by dpgraham**
Can't do 'beta' releases because it tries to prune shrinkwrap when `npm-shrinkwrap.json` doesn't exist.

This checks that `npm-shrinkwrap.json` exists before pruning dependencies. If it doesn't exist, don't prune.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,6 +114,7 @@
     "gulp": "^4.0.0",
     "handlebars": "^4.0.10",
     "mocha": "^6.0.0",
+    "path-exists-cli": "^1.0.0",
```
Comment (jlipps) [inline package.json]: i hate that we need another dev dep just to do a `ls` but, sigh, cross platform

**Review by jlipps — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (imurchie) [reply package.json]: Should be able to be `test -e npm-shrinkwrap.json && (...)`

Comment (dpgraham) [reply package.json]: Would that work in windows @imurchie ?

Comment (imurchie) [reply package.json]: No idea. But given the usage of this, is it necessary to be cross-platform?

Comment (dpgraham) [reply package.json]: @imurchie I'll change it to that since we only publish from Mac

**Pull request merged**